l10n_it_fatturapa_out/data/invoice_it_template.xml

```diff
@@ -439,10 +439,12 @@ e 'line' per riga di fattura (a seconda del livello in cui sono chiamati)
                         t-set="id_fiscale_iva"
                         t-value="get_id_fiscale_iva(partner_id.commercial_partner_id)"
                     />
+                    <t t-if="partner_id.commercial_partner_id.vat">
```

Comment:
Non so a naso preferirei una cosa del tipo:
```
<IdFiscaleIVA t-if="id_fiscale_iva['id_codice']">
...
</IdFiscaleIVA>
```

e modificare `get_id_fiscale_iva()` per restituire vuoto invece di 99999999 nel caso in oggetto.

Altrimenti se non c'è né IVA né c.f. generiamo un xml non valido...

Reply:
https://github.com/OCA/l10n-italy/pull/2973/commits/4188e627baf0c37538b8a1894da04950bd8424d8 ho sistemato fammi sapere se va bene e poi unisco i commit.